src/modules/routes/index.ts

```diff
@@ -69,17 +70,38 @@ async function register(server: Server) {
     handler: addDailyVisitsController,
   });
 
+  server.route({
+    method: 'GET',
+    path: '/expeditions/weekly-rewards',
+    options: {
+      description: `Get an address's weekly rewards state for given address`,
+      validate: {
+        query: {
+          address,
+        },
+      },
+      tags: [
+        'api',
+        'expeditions',
+        'weekly liquidity',
+        'weekly rewards',
+        'weekly rewards fragments',
+      ],
+    },
+    handler: getWeeklyRewardsFragmentsState,
+  });
+
   server.route({
     method: 'GET',
     path: '/expeditions/weekly-liquidity',
     options: {
-      description: `Get an address's weekly liquidity`,
+      description: `Get an address's weekly liquidity rewards state for given address`,
       validate: {
         query: {
           address,
         },
       },
-      tags: ['api', 'expeditions', 'weekly liquidity'],
+      tags: ['api', 'expeditions', 'weekly liquidity', 'weekly rewards'],
     },
     handler: getWeeklyLiquidityPositionDeposits,
   });
```

Comment:
I think we can remove this route as `/expeditions/weekly-rewards` have both liquidity provision and staking